### PR TITLE
Add material ui with dark mode

### PR DESCRIPTION
**Pull request by cstegeman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
   "private": true,
   "type": "module",
   "dependencies": {
+    "@emotion/react": "^11.6.0",
+    "@emotion/styled": "^11.6.0",
```
Comment (abhiofficial66) [inline package.json:8]: You are probably using this package for styled components. You could also use `styled-components` instead.

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
```
Comment (abhiofficial66) [inline src/index.js:9]: Might be out of scope for now, but it would be cool to have linting in place to allow single quotes instead of double quotes.

**Comment by abhiofficial66** [comment]
Also did not mention it yesterday, setting up repo with create react app is also has a nice learning curve specially when you want to modify webpack config and the magic you need to do achieve that.

File: package.json (hunk 1 above)
Comment (cstegeman) [reply package.json:8]: The new material-ui version specifically required this package. Otherwise it doesn't compile

File: src/index.js (hunk 2 above)
Comment (cstegeman) [reply src/index.js:9]: Good one, will put that on my todo list!

Comment (cstegeman) [reply src/index.js:9]: + prettier config

**Comment by cstegeman** [comment]
> Also did not mention it yesterday, setting up repo with create react app is also has a nice learning curve specially when you want to modify webpack config and the magic you need to do achieve that.

I am assuming you mean **without** create-react-app? Create react app has an option to convert it to go with the out of the box tools

**Pull request merged**